index.html

```diff
@@ -128,7 +128,7 @@ <h2>Sponsors</h2>
 	<footer>
 		This site is maintained by the <a href="//github.com/visjs">visjs community</a>. 
 		Help to <a href="//github.com/visjs/www.visjs.org/edit/master/index.html">improve this page</a>!
-		<a class="license" rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/80x15.png" /></a>
+		<a class="license" rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/"> <br> <img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-sa/4.0/80x15.png" /></a>
```

Comment:
What is the purpose of this?

Reply:
This <br> tag is for mobile responsive , it's style property is { display: none ;} , but for viewport device width less than 746px it's style properties is set to {display: visible ;}. So that the license image in the footer arranges itself in the new line for the specific device width.

Reply:
Okay but this raises even more questions. Why is the line break itself a link? What about the magic number 746?

Reply:
> I would also prefer if the buttons (view examples, view docs…) stayed in the same arrangment all the time (it doesn't look good when 4 are on the first line and only one on the second)

I will surely try to implement the specific changes you require. Thank you.

Reply:
> Why is the line break itself a link?
I express regret for this mistake of mine , of putting the br tag before the line . I will resolve the mistakes in the next PR .

Reply:
Why in the next PR? Why not here?

PS: Just in case your new to PRs you can just push more commits to the same branch and they'll appear here.

Reply:
> What about the magic number 746?

I am unable to get you , what exactly you are trying to refer .

Reply:
There is `max-width: 746px`. Why 746?

Reply:
Yes I am new to PRs and thanks for helping me.

Reply:
No problem, everyone was new at some point.

Reply:
> Why 746?

I observed the breakpoints from responsive design mode , at 746px the content aligned itself to different view .

Reply:
This is highly dependent on the browser used and can easy fall apart even with an update of the same browser.

Reply:
I have observed the wide range of device width around this range 300px - 1000px and observed each breakpoints. This was the best number I was able to get. Can you suggest some more accurate tools.

Reply:
I suggest styling `<main>` at fixed width as the sections already are. In other words if the screen is wide enough to accommodate 3 columns make the main 3 times wider than one section, if only two then 2 times and if only one then 1 time. You can then center the main with `margin: 0 auto;` to center the whole thing.